src/main/java/core/basesyntax/FigureDrawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawable {
+    void getInfo();
```

Comment:
Don't use verbs for class/interface names. Consider renaming the interface to something that represents an entity or a characteristic, such as `Drawable` instead of `FigureDrawable`.